certified-connectors/OrbusInfinity/apiProperties.json

```diff
@@ -0,0 +1,70 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oAuthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "scopes": [
+            "user_impersonation"
+          ],
+          "redirectMode": "GlobalPerConnector",
+          "properties": {
+            "IsFirstParty": "False",
+            "AzureActiveDirectoryResourceId": "{resourceUri}",
+            "IsOnbehalfofLoginSupported": true
+          },
+          "customParameters": {
+            "LoginUri": {
+              "value": "https://login.microsoftonline.com"
+            },
+            "TenantId": {
+              "value": "common"
+            },
+            "EnableOnbehalfOfLogin": {
+              "value": false
+            }
+          },
+          "clientId": "<<Please add your clientID here>>"
+        },
+        "uiDefinition": null
```

Comment:
Please fix the below error, kindly refer this schema and update https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/paconn-apiProperties.schema.json

@{level=Critical; message=Invalid type for the UIDefinition object.; valuePath=$.properties.connectionParameters.token.uiDefintion; value=}

Reply:
fixed